projects/1.20/assets/tropicraft/tropicraft/lang/zh_cn.json

```
@@ -0,0 +1,548 @@
+  "item.tropicraft.yellow_scuba_goggles": "黄色潜水镜",
+  "item.tropicraft.yellow_scuba_harness": "黄色潜水背带",
+  "item.tropicraft.yellow_umbrella": "黄色伞",
+  "item.tropicraft.zircon_axe": "锆石斧",
+  "item.tropicraft.zircon_gem": "锆石宝石",
+  "item.tropicraft.zircon_hoe": "锆石锄",
+  "item.tropicraft.zircon_pickaxe": "锆石镐",
+  "item.tropicraft.zircon_shovel": "锆石锹",
+  "item.tropicraft.zircon_sword": "锆石剑",
+  "item.tropicraft.zirconium_axe": "锆斧",
+  "item.tropicraft.zirconium_gem": "锆宝石",
+  "item.tropicraft.zirconium_hoe": "锆锄",
+  "item.tropicraft.zirconium_pickaxe": "锆镐",
+  "item.tropicraft.zirconium_shovel": "锆锹",
+  "item.tropicraft.zirconium_sword": "锆剑",
+  "itemGroup.tropicraft.tropicraft": "热带世界",
+  "scuba.tropicraft.air_time": "剩余空气：%s",
+  "scuba.tropicraft.depth": "当前深度：%s",
+  "scuba.tropicraft.dive_time": "潜水时间：%s",
+  "scuba.tropicraft.max_depth": "最大深度：%s",
+  "scuba.tropicraft.scuba.visibility": "水下雾减少",
+  "tropicraft.container.bambooChest": "竹箱子",
+  "tropicraft.container.bambooChestDouble": "大型竹箱子",
+
+  "block.tropicraft.blue_flowering_leaves": "盛开的蓝色花叶",
+  "block.tropicraft.purple_flowering_leaves": "盛开的紫色花叶",
+  "block.tropicraft.red_flowering_leaves": "盛开的红色花叶",
+  "block.tropicraft.white_flowering_leaves": "盛开的白色花叶",
+  "block.tropicraft.yellow_flowering_leaves": "盛开的黄色花叶",
+
+  "block.tropicraft.eel_grass": "海菖蒲",
+  "block.tropicraft.eel_grass_block": "海菖蒲块",
+  "block.tropicraft.fern_seagrass": "喜盐草",
+  "block.tropicraft.fern_seagrass_block": "喜盐草块",
+  "block.tropicraft.flowering_eel_grass": "盛开的海菖蒲",
+  "block.tropicraft.flowering_tall_eel_grass": "盛开的高海菖蒲",
+  "block.tropicraft.jigarbov_acacia_wall_torch": "Jigarbov金合欢木红石火把",
+  "block.tropicraft.jigarbov_birch_wall_torch": "Jigarbov白桦木红石火把",
+  "block.tropicraft.jigarbov_black_mangrove_wall_torch": "Jigarbov黑色红树木红石火把",
+  "block.tropicraft.jigarbov_dark_oak_wall_torch": "Jigarbov深色橡木红石火把",
+  "block.tropicraft.jigarbov_jungle_wall_torch": "Jigarbov丛林木红石火把",
+  "block.tropicraft.jigarbov_light_mangrove_wall_torch": "Jigarbov浅色红树木红石火把",
+  "block.tropicraft.jigarbov_oak_wall_torch": "Jigarbov橡木红石火把",
+  "block.tropicraft.jigarbov_red_mangrove_wall_torch": "Jigarbov红树木红石火把",
+  "block.tropicraft.jigarbov_spruce_wall_torch": "Jigarbov云杉木红石火把",
+  "block.tropicraft.matted_eel_grass": "海菖蒲覆层砂",
+  "block.tropicraft.matted_fern_seagrass": "喜盐草覆层砂",
+  "block.tropicraft.matted_noodle_seagrass": "针叶藻覆层砂",
+  "block.tropicraft.matted_sickle_seagrass": "木秆藻覆层砂",
+  "block.tropicraft.noodle_seagrass": "针叶藻",
+  "block.tropicraft.noodle_seagrass_block": "针叶藻块",
+  "block.tropicraft.palm_button": "棕榈木按钮",
+  "block.tropicraft.portal_water": "热带世界传送门水",
+  "block.tropicraft.sickle_seagrass": "木秆藻",
+  "block.tropicraft.sickle_seagrass_block": "木秆藻块",
+  "block.tropicraft.tall_eel_grass": "高海菖蒲",
+  "block.tropicraft.tall_fern_seagrass": "高喜盐草",
+  "block.tropicraft.tall_sickle_seagrass": "高木秆藻",
+  "block.tropicraft.teleport_water": "热带世界传送门",
```

| **审阅级别** |
|:--------------:|
|❓ **疑问** / 不阻止合并|

原文：**Teleport Water**
```suggestion
  "block.tropicraft.teleport_water": "热带世界传送门",
```
为什么

---

因为传送门介质就是水，泡在这个水里面就会传送到该维度